spec/gateways/govuk_organisations_register_gateway_spec.rb

```diff
@@ -0,0 +1,25 @@
+describe Gateways::GovukOrganisationsRegisterGateway do
```

Comment:
I know this is pre-existing behaviour, but I don't like these specs relying on the behaviour of a docker container - feels like the set-up for this test is very distant to the assertion, which could be confusing for a reader.

I like the idea of the docker container for supporting local dev, but I'm not sure about it for automated testing. I know other people at Made Tech have used a similar strategy, be interested to hear how it panned out for them long-term.

This PR isn't a regression - this paradigm already exists in the codebase - so I don't want to block us moving forward. 
 Just wanted to share the thought out loud.

Reply:
And of course, the _moment_ I hit submit, I work out that they don't rely on the docker container at all, they rely on something in `spec_helper`.

That's better, but still suffering from the same problem - the setup is distant from the assertion. I think this could be pretty cleanly addressed with the use of `shared_context` - if that context is even for anything but this single test?

Reply:
Yeah i hear you @kjdchapman.
Let's try move this setup closer to the assertions after we get this in.